ocp-scripts/clone-project.sh

```diff
@@ -88,14 +88,13 @@ echo "Provided params: \
 SOURCE_PROJECT="$PROJECT_ID-$SOURCE_ENV"
 TARGET_PROJECT="$PROJECT_ID-$TARGET_ENV"
 
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 echo "[INFO]: creating workplace: mkdir -p oc_migration_scripts/migration_config"
 mkdir -p oc_migration_scripts/migration_config
 cd oc_migration_scripts
 echo $(pwd)
-export_url="https://$BITBUCKET_HOST/projects/opendevstack/repos/ods-core/raw/ocp-scripts/export-project.sh?at=refs%2Fheads%2Fproduction"
-curl --fail -s --user $CREDENTIALS -G $export_url -d raw -o export.sh
-import_url="https://$BITBUCKET_HOST/projects/opendevstack/repos/ods-core/raw/ocp-scripts/import-project.sh?at=refs%2Fheads%2Fproduction"
-curl --fail -s --user $CREDENTIALS -G $import_url -d raw -o import.sh
+cp -v "$SCRIPT_DIR/export-project.sh" .
+cp -v "$SCRIPT_DIR/import-project.sh" .
```

Comment:
I think these can be removed. I only see two ways this script can be used:

1. It is curled. Then the other files are also curled and all files are in the same folder. Just calling `sh export-project.sh` will work.

2. It is not curled, but cloned via Git. Then the other files are in the same folder as well, which means just calling `sh export-project.sh` will work too.

Reply:
I like your proposed solution with the clone-script checking itself for the presence of export import scripts and curling as needed.

Reply:
I tested this version in our environment including for backwards compatibility.